torch/ao/quantization/experimental/APoT_tensor.py

```diff
@@ -2,11 +2,13 @@
 from torch.ao.quantization.experimental.quantizer import APoTQuantizer
 
 # class to store APoT quantized tensor
-class TensorAPoT(torch.Tensor):
+class TensorAPoT():
```

Comment:
why is TensorAPoT no longer inheriting from torch.Tensor?

Reply:
When TensorAPoT inherits from torch.Tensor I get an issue with my TensorAPoT `init` function since I am not passing in the expected args:
`TypeError: new() received an invalid combination of arguments - got (Tensor, int, int, bool), but expected one of:
 * (*, torch.device device)
 * (torch.Storage storage)
 * (Tensor other)
 * (tuple of ints size, *, torch.device device)
 * (object data, *, torch.device device)`

I didn't think any of these made sense for my TensorAPoT object since I am passing in a quantizer object. Closest was the `(Tensor other)` option, but in that case I would need to create the quantizer in my `init` function and would need to pass in the args needed by quantizer `init` , which would also violate the expected args.

Is it okay if TensorAPoT doesn't inherit from torch.Tensor?

Reply:
I feel this is fine for now, this might involve a bit more design about how do we allow people to extend quantized tensor in python, that's something we can explore in H2